routemaster/client/assertion_helpers.rb

```diff
@@ -0,0 +1,18 @@
+require 'uri'
+
+module Routemaster
+  class Client
```

Comment:
Im not sure what kind of issue this would cause if we are not using autoloading that would be solved by changing the `Client` to a module. Could you give an example?